package.json

```diff
@@ -24,6 +24,7 @@
     "ag-grid-react": "~21.1.0",
     "classnames": "~2.2.6",
     "clipboard": "~2.0.4",
+    "clipboard-copy": "^3.0.0",
```

Comment:
@lbwexler / @haynesjm42  - saw the thread re. avoiding the two libraries, but doesn't look like the old one was actually removed... 😀OK to actually take it out?

Reply:
great catch on this one.  Removed!